Testing/functional/tests/cypress/integration/ui/pages/LandingPage.js

```diff
@@ -116,4 +116,10 @@ describe("Landing Page", () => {
         cy.get("#menuBtnLogin").should("not.exist");
         cy.get("footer > .navbar").should("not.visible");
     });
+
+    it("Validate active tiles", () => {
+        cy.get("[data-testid=active-tile-ClinicalDocument]").should(
+            "be.visible"
+        );
+    });
```

Comment:
Since this is a UI test, we should use enableModules to check that the tile appears when the ClinicalDocument module is enabled and doesn't appear when it's disabled.